cylc/flow/task_job_mgr.py

```diff
@@ -25,52 +25,46 @@
 """
 
 import json
-from logging import DEBUG, CRITICAL, INFO, WARNING
 import os
+from copy import deepcopy
```

Comment:
Looks like I have accidentally alphabetized these - me being a bit overzealous with the auto format on vscode. Happy to revert this if it makes reviewing easier, although I'm not sure if it looks tidier that way?

Reply:
"alphabetizing" +1
"multiple imports per line" -1

Reply:
> "multiple imports per line" / "right-hand code squash" -1
I'll fix these, thanks!